.travis.yml

```diff
@@ -74,13 +74,33 @@ matrix:
       <<: *_android
 
 before_install:
-  - if [[ "$PLATFORM" =~ android ]]; then nvm install $TRAVIS_NODE_VERSION; fi # manually install Node for `language: android`
+  # manually install Node for `language: android`
+  - if [[ "$PLATFORM" =~ android ]]; then nvm install $TRAVIS_NODE_VERSION; fi 
   - node --version
   - if [[ "$PLATFORM" =~ android ]]; then gradle --version; fi
   - if [[ "$PLATFORM" =~ ios ]]; then npm install -g ios-deploy; fi
   - npm install -g cordova
+  # install paramedic if not running on paramedic repo
+  - if ! [[ "$TRAVIS_REPO_SLUG" =~ cordova-paramedic ]]; then npm install -g github:apache/cordova-paramedic; fi 
```

Comment:
Do we have an issue for a new release/issue to remove the `github:apache/` aprt?

This should be added as a reminder that the installation is preferred via npm registry rather than github source. (once we've [finally] managed to publish a new version after 4 years)

Reply:
No, but you may create one. 

The benefit of this is that the plugin tests are _not_ dependent on (npm) release but a a commit to master will affect the plugins instantly.
A disadvantage is that a commit to master will affect the plugins instantly - potentially breaking them ;)